governance/reviews/cloudevents-08-29-2023.md

```diff
@@ -0,0 +1,210 @@
+
+Freshness: CloudEvents has updated their governance docs multiple times in the project history.  The last batch of governance updates was 8 months ago.
+
+#### Governance Drift
+
+<!---  Are the governance activities being carried out? Are community meetings (if any) happening? Are required elections and votes taking place? Are official communications channels accessible, staffed and responsive? Are they being used? Are questions and proposed updates/changes to governance (if any) being transparently discussed and addressed? -->
+
+Governance Drift can occur when the executed and observable governance of a project deviates from the documented governance of the project.
+
+Governance drift for CloudEvents cannot be determined because the written governance is too hard to decipher.
+
+#### Ownership
+
+The main project uses OWNERS files which appear to match declare roles elsewhere.  At this time, we cannot audit actual Github permissions.
+
+The SDKs do not appear to have any controls on permissions, and as such are likely very out of sync with project roles.
+
+### Maintainer List(s)
+
+CloudEvents' three types of leaders need to be evaluated separately.
+
+#### Admins
+
+The list of Admins is current, and appears well-balanced, with four admins representing four different organizations.  However, admins are listed only by GH handle, with no affiliation, contact information, or history; listing full information on each admin would enhance transparency.  The Admins were set up as roles-for-life, with only one change of admin since the project was founded in 2018.  There are no defined processes for removing an admin for inactivity, nor any expectation that new Admins will be recruited.
+
+Since, at this time one of the four admins (Ken Owens) is no longer active in the project per Devstats and meeting tracking, it would benefit Cloud Events to document a full Admin lifecycle including recruitment, promotion, qualifications, and emeritus status.  Documenting this and following it would help ensure that the project retains a continuously active leadership.
+
+#### Voting Members
+
+In contrast to the admins, voting members are defined by their activity through participation in spec meetings, ensuring that the list of voting members is always relatively current.  This makes it quite an easy route to get involved in the project, and prevents staleness.  The list of voting members is tracked through a [spreadsheet](https://docs.google.com/spreadsheets/d/1bw5s9sC2ggYyAiGJHEk7xm-q2KG6jyrfBy69ifkdmt0/edit?pli=1#gid=0), which raises some issues around transparency and auditability (for example, who owns that spreadsheet?).  Balance is ensured by voting rules which do not permit employees of the same company from voting more than once.
```

Comment:
Our governance doc now says: An Admin will be responsible for updating the spreadsheet.

Reply:
Where's the spreadsheet stored?  In whose account?  The Projects staff can help you with putting it in a CNCF-owned account.  We really don't want it to get deleted if someone changes jobs.